### PR TITLE
finish pango

### LINKED ISSUE #19: support multiple windows
- `Gl` is bound to a window so we probably need to create multiple `Renderer`s and `api`s (no change to what we do currently)
- resources have to be uploaded to each specific window (on demand) which might not be so easy, especially with fonts (different sizes, weights, ...)

### LINKED ISSUE #52: add textAlign
- add `textAlign` to `TextContainer.update()` props
- compute width of each line
- compute maxWidth of all lines
- move glyphs proportionally (maxWidth - lineWidth)

or, it could be done in one pass but we would have to maintain offset, relative to max & previous line, including negative values and after that, move all glyphs at once (SIMD?)

### PR DESCRIPTION
**Pull request by cztomsik**
close #52 
close #19

### CODE REVIEW

**Comment by cztomsik** [comment]
rebased against new codegen

**Comment by cztomsik** [comment]
finally got multi-window working
image

File: README.md [hunk 1]
```diff
@@ -55,7 +55,17 @@ const styles = StyleSheet.create({
...
-We use [webrender](https://github.com/servo/webrender) for the actual drawing, yet there is **no DOM**, so we can be faster and yet use (much) less memory. [Read more here](./docs/webrender.md)
+## Why it's interesting
+- quick to setup, apart from rust & few libs, it should be just one `npm install` away
+- can be combined with most of the libraries you already know (react, mobx, lodash, ...)
+- works with existing tooling (debug in vscode, profile in chrome devtools, react-devtools, ...)
+- hot-reload works even without webpack (and it's faster)
+- bundle can be made using already established and mature tools (ncc + electron-builder)
+- low memory footprint (when compared to electron)
+- the language & platform you already know (when compared to flutter)
+- intention to support most of the CSS properties (when compared to react-native)
```
Comment (twop) [inline README.md]: Not sure if im fully onboard with that statement :(

**Comment by cztomsik** [comment]
image

**Comment by cztomsik** [comment]
relates to #7

**Pull request merged**